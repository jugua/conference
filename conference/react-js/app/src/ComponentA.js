import React from 'react';

const Foo = React.createClass({
  render() {
    return (
      <p>I am not a very smart component...</p>
    );
  }
});

export default Foo;
