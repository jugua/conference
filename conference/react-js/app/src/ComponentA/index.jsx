import React from 'react';
import './component.sass';

export default class extends React.Component {
	constructor(props) {
		super(props);
	}

	render() {
		return (
			<h2>COMPONENT A - Folkbit</h2>
		);
	}
}
