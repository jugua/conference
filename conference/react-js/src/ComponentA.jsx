import React from 'react';

const Foo = React.createClass({
  render() {
    return (
      <p>I am not a very smart component...ssss</p>
    );
  },
});

export default Foo;
