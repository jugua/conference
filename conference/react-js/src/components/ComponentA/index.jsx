import React from 'react';
import './component.sass';

export default class MyComponent extends React.Component {
	constructor(props) {
		super(props);
	}

	render() {
		return (
			<p>I am not a very smart component...</p>
		);
	}
}
