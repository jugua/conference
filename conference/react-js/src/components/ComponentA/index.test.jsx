import React from 'react';
import renderer from 'react-test-renderer';

import ComponentA from './index';

jest.unmock('./index');

describe('ComponentA', () => {
  it('ComponentA renders correctly', () => {
    const tree = renderer.create(
      <ComponentA />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
