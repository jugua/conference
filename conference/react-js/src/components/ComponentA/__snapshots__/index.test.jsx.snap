// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ComponentA ComponentA renders correctly 1`] = `
<p>
  I am not a very smart component...dsdsdsd
</p>
`;
